File: problems/r/r2/r2.py
#! /usr/bin/python3
"""
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
"""

def main():
    r1, s = [int(x) for x in input().split()]
    r2 = 2 * s - r1
    print(r2)

if __name__ == '__main__':
    main()

File: problems/r/r2/r2.c
/*
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
*/

#include <stdio.h>
#include <stdlib.h>

int main(void) {
    int r1, r2, s;
    while (scanf("%d%d", &r1, &s) == 2)
        r2 = 2 * s - r1;
        printf("%d", r2);
    return 0;
}